Ginger/Ginger/UserControlsLib/UCTreeView/TreeView1.xaml.cs

```diff
@@ -108,22 +109,62 @@ private void xTreeViewTree_ItemSelected(object sender, EventArgs e)
         }
 
         private void xSearchTextBox_TextChanged(object sender, TextChangedEventArgs e)
-        {
+        {             
             if (xSearchTextBox.Text.Length > 0)
             {
                 xSearchNullText.Visibility= Visibility.Collapsed;
                 xSearchClearBtn.Visibility = Visibility.Visible;
             }
             else
-            {
-                xSearchNullText.Visibility = Visibility.Visible;
-                xSearchClearBtn.Visibility = Visibility.Collapsed;
+            {                                                
+                List<TreeViewItem> pathNodes = new List<TreeViewItem>();
```

Comment:
We do need
xSearchNullText.Visibility = Visibility.Visible;
xSearchClearBtn.Visibility = Visibility.Collapsed;